Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax = docker/dockerfile:1
 
 # Adjust NODE_VERSION as desired
-ARG NODE_VERSION=20.8.1
+ARG NODE_VERSION=22.13.1
 FROM node:${NODE_VERSION}-slim AS base
```

Comment:
Let's simplify this as follows - it will enable Dependabot to automatically open PRs to update the Node.js version with future releases.

```
FROM node:22.13.1-slim AS base